.github/workflows/go.yaml

```diff
@@ -152,6 +152,7 @@ jobs:
       if: ${{ github.event_name == 'schedule' }}
       run: |
         echo "IMAGE_TAG=nightly" | tee -a ${GITHUB_ENV}
+        echo "HELM_CHART_VERSION_SUFFIX=-${IMAGE_TAG}" | tee -a ${GITHUB_ENV}
```

Comment:
the env var isn't available here yet (needs next step) but you could make it so:
```
IMAGE_TAG=nightly
echo "IMAGE_TAG=${IMAGE_TAG}" | tee -a ${GITHUB_ENV}
echo "HELM_CHART_VERSION_SUFFIX=-${IMAGE_TAG}" | tee -a ${GITHUB_ENV}
```